src/lib.rs

```diff
@@ -836,6 +838,28 @@ impl UmberWm {
                     let map_notify: &xcb::MapNotifyEvent = unsafe { xcb::cast_event(&event) };
                     self.setup_new_window(map_notify.window()).ok();
                 }
+                if r == xcb::PROPERTY_NOTIFY as u8 {
+                    let property_notify: &xcb::PropertyNotifyEvent = unsafe { xcb::cast_event(&event) };
+                    println!("property_notify: {:x} {}", property_notify.window(), property_notify.atom());
+                    let cookie = xcb::get_property(
+                        &self.conn,
+                        false,
+                        property_notify.window(),
+                        property_notify.atom(),
+                        xcb::ATOM_ANY,
+                        0,
+                        1024,
+                    );
+                    if let Ok(reply) = cookie.get_reply() {
+                        if let Ok(res) = std::str::from_utf8(reply.value()) {
+                            let n = property_notify.atom();
+                            let na = xcb::get_atom_name(&self.conn, n).get_reply();
+                            let nam = na.unwrap();
+                            let name = nam.name();
```

Comment:
Why not this?
```rust
let name = xcb::get_atom_name(&self.conn, n).get_reply().unwrap().name();
```

Reply:
I have to keep the last line at least, otherwise I get "temporary value dropped while borrowed"

Reply:
Oh one of those lol. Got it.